src/utils/metaform-utils.ts

```diff
@@ -435,6 +435,60 @@ namespace MetaformUtils {
     return moment.duration(Math.floor(totalTime / replyCount));
   };
 
+  /**
+   * Create status section for new metaform
+   * 
+   * @returns status section for form
+   */
+  export const formStatusSection = (): MetaformField => {
```

Comment:
```suggestion
  export const createFormStatusField = (): MetaformField => {
```
Not part of this PR but function names should contain verbs and also this function is returning a MetaformField, not MetaformSection.